stl/inc/variant

```diff
@@ -962,18 +962,16 @@ using _Variant_destroy_layer = conditional_t<conjunction_v<is_trivially_destruct
 #pragma clang diagnostic ignored "-Wdeprecated-volatile"
 #else // ^^^ Clang / not Clang vvv
 #pragma warning(push)
+#pragma warning(disable : 4242) // '%s': conversion from '%s' to '%s', possible loss of data

```

Comment:
can/should we disable this in one pragma warning?

Reply:
It's possible to disable multiple warnings in a single pragma, and they do share the exact same message (at this time), but stylistically I prefer one warning per line. This makes it clear that they do share the same message, and that we haven't grouped different warnings together accidentally.

Reply:
I could possibly be convinced that we need some additional annotation here - and probably elsewhere where these are suppressed - to indicate that the messages really are identical. It does look like copypasta to the uninitiated.

Reply:
Do you want to update this PR, or file a `documentation` issue? I haven't yet bundled this up into an MSVC-PR.

Reply:
Pushed an update to append `(Yes, duplicated message.)` to suppressions for 4244 that appear after suppressions for 4242. It's not as clear as it could be, but does all fit on one line.